tasks/actorstate/miner.go

```diff
@@ -31,6 +33,31 @@ func init() {
 	}
 }
 
+func ModelExtractors(m model.Persistable) (func(ctx context.Context, ectx *MinerStateExtractionContext) (model.Persistable, error), error) {
```

Comment:
In a subsequent PR this switch can be replaced with a map by forcing models to register themselves with a task: https://github.com/filecoin-project/sentinel-visor/blob/frrist/tasksv2/tasks/actorstate/miner/tasks/extractors/currentdeadlineinfo.go#L14 is a WIP